### ISSUE TITLE
Investigate Usage of Self-Hosted Renovate Runner for Registry

### ISSUE THREAD

**Issue by thepetk** [devfile/api#1202]
## Which area/kind this issue is related to?

/area ci
/area registry

## Issue Description

As part of https://github.com/devfile/api/issues/1133 there is an option to investigate the usage of a self hosted bot for renovate instead of using the mend.io runner. The renovate runner can be found here: https://github.com/platform-engineering-org/renovate-runner. As a result it would be good to try this renovate runner against a forked repo of devfile registry.

**Target date**: 30 July 2023

**Comment by thepetk** [comment]
I'll try to test the self hosted runner against my registry fork in order to check the differences between mend.io and the self hosted format.

**Comment by thepetk** [comment]
I've added a [PR](https://github.com/platform-engineering-org/renovate-runner/pull/38) in the self-hosted runner repo in order to include my fork of the registry repo. If this is merged we can see in action the results for this automation

**Comment by thepetk** [comment]
The PR was merged from the platform engineering team and we have the first run here: https://github.com/platform-engineering-org/renovate-runner/actions/runs/5691571384/job/15426998337

I've added to our renovate.json the shared config to check how renovate behaves with shared and custom config.

In general, I've noticed some things changing:
* The self hosted config provides a dependency dashboard (an issue created like https://github.com/thepetk/registry/issues/53). There we can have an overview of all updates handled by renovate-bot. Ofc, this feature is available to the mend.io runner too.

* In the shared config we merge all changes to one PR (so there will be one pr with all registry image updates). I'd say for all other repos is fine to have this behavior, but as registry updates demand a lot of testing, it might be good to have one PR per image update.
